src/modules/previewpane/README.md

```diff
@@ -1,28 +1,26 @@
-<center>
-
 <img width="200" src="../../../doc/images/Logo.jpg">
 
-# PowerPreview
+# File Explorer
+
+File Explorer add ons right now are just limited to Preview Pane additions for File Explorer.
```

Comment:
I'm not a native English speaker, so I may be wrong: "add ons" -> "add-ons"
Also add commas: "File Explorer add-ons, right now, are just limited"